lib/l10n/intl_en.arb

```diff
@@ -190,16 +192,33 @@
   "flutter_website": "https://flutter.dev",
 
   "skip_discovery": "Skip tour",
-  "discovery_navbar_dashboard_page_title": "Dashboard",
-  "discovery_navbar_dashboard_page_details": "The Dashboard page groups together the most frequently consulted information, in the form of cards.\n\nIf you keep your finger on the card, you can move it up and down to rearrange the cards.\n\nTo delete a card, swipe it to the left or right.",
-  "discovery_navbar_schedule_page_title": "Schedule",
-  "discovery_navbar_schedule_page_details": "The Schedule page allows you to view the events of your school calendar.",
-  "discovery_navbar_student_page_title": "Student",
-  "discovery_navbar_student_page_details": "The Student page displays your grades and profile information.",
-  "discovery_navbar_ets_page_title": "ÉTS",
-  "discovery_navbar_ets_page_details": "The ÉTS page allows you to quickly consult all of the important ÉTS links, such as Moodle, MonÉTS and security measures.",  
-  "discovery_navbar_more_page_title": "More",
-  "discovery_navbar_more_page_details": "The More page gives you access to additional options of the application.",
+
+  "discovery_navbar_dashboard_title": "Dashboard",
+  "discovery_navbar_dashboard_details": "The Dashboard page groups together the most frequently consulted information, in the form of cards.\n\nIf you keep your finger on the card, you can move it up and down to rearrange the cards.\n\nTo delete a card, swipe it to the left or right.",
+  "discovery_page_dashboard_restore": "To restore the changes, click on the icon at the top right.",
+  
+  "discovery_navbar_schedule_title": "Schedule",
+  "discovery_navbar_schedule_details": "The Schedule page allows you to view the events of your school calendar.",
+  "discovery_page_schedule_settings": "In the settings, you can change the calendar format and the first day of the week displayed.",
+  
+  "discovery_navbar_student_title": "Student",
+  "discovery_navbar_student_details": "The Student page displays your grades and profile information.",
+  
+  "discovery_page_student_grades_session": "Your courses are divided by session, with the most recent at the top. At any moment, you can drag the list to refresh it.",
+  "discovery_page_student_grades_grade_button": "Here you see your current grades. By tapping on it, you will be able to see the details of all the evaluations obtained in the course.",
```

Comment:
More of an open question, but should we do a discovery on the page grade details? @camillebrulotte @apomalyn @AlexandreMarkus

Reply:
I do agree with you :smile:

Reply:
@apomalyn @MysticFragilist Enfait vous pensiez à mettre quoi comme description dans la grade details ? :) 

Autre que les cercles de progress bar qui pourraient être ambigu et les evaluation tile, je sais pas trop quoi expliquer d'autre.